node/src/components/chainspec_loader/chainspec.rs

```diff
@@ -434,16 +441,203 @@ mod tests {
         assert_eq!(spec.genesis.deploy_config.block_max_deploy_count, 125);
         assert_eq!(spec.genesis.deploy_config.block_gas_limit, 13);
 
-        assert_eq!(spec.genesis.costs.regular, 13);
-        assert_eq!(spec.genesis.costs.div, 14);
-        assert_eq!(spec.genesis.costs.mul, 15);
-        assert_eq!(spec.genesis.costs.mem, 16);
-        assert_eq!(spec.genesis.costs.initial_mem, 17);
-        assert_eq!(spec.genesis.costs.grow_mem, 18);
-        assert_eq!(spec.genesis.costs.memcpy, 19);
-        assert_eq!(spec.genesis.costs.max_stack_height, 20);
-        assert_eq!(spec.genesis.costs.opcodes_mul, 21);
-        assert_eq!(spec.genesis.costs.opcodes_div, 22);
+        assert_eq!(
+            spec.genesis.wasm_config,
+            WasmConfig {
```

Comment:
Let's construct this value outside of the assertion and bind it to a name like expected_costs.